config/php/fpm/standardebooks.org.ini

```diff
@@ -10,6 +10,9 @@ allow_url_fopen = false
 allow_url_include = false
 expose_php = Off
 
+post_max_size = 128M
+upload_max_filesize = 128M
```

Comment:
Given the conversation so far, I vote for these values (here and below):

```
post_max_size = 64M
upload_max_filesize = 32M
```

That will restrict the total post data to 64M and each individual file to 32M. 

1. If the user uploads a file above 64M, they will lose their form values. Too bad. They will at least get an error, thanks to your new code.
2. If the user uploads a file between 32M and 64M, they will get an error, and the form will preserve their values, so that's nice.